### DISCUSSION TITLE
JSON parser support

### DISCUSSION THREAD

**Discussion by taran0505** [alecthomas/participle#383] [Q&A]
Does the package provides the support for creating a JSON parser?
How to parse array of object in that case ?

```
{
  "details": {
        "person": [
            {
                "name": "ABC",
                "age": 23
            },
            {
                "name": "XYZ",
                "age": 28
            }
        ]
    }
 }
```

**Comment by alecthomas** [comment]
Participle is a library for _creating_ parsers, not for providing specific parsers. That said, the included examples include a [TOML](https://github.com/alecthomas/participle/tree/master/_examples/toml) parser, which has array support.

**Comment by zhuliquan** [comment]
> Does the package provides the support for creating a JSON parser? How to parse array of object in that case ?
> 
> ```
> {
>   "details": {
>         "person": [
>             {
>                 "name": "ABC",
>                 "age": 23
>             },
>             {
>                 "name": "XYZ",
>                 "age": 28
>             }
>         ]
>     }
>  }
> ```

I implemented it, see PR https://github.com/alecthomas/participle/pull/395
